README.md

```diff
@@ -1,3 +1,161 @@
 # FairGBM
 
-_Under construction; will be released in mid September._
+> **Under Construction**; release date: 16th of September.
+
+FairGBM is an easy-to-use and lightweight fairness-aware ML algorithm.
+Specifically, it enables **constrained optimization** with Gradient Boosting models.
+
+FairGBM builds upon the popular [LightGBM](https://github.com/microsoft/LightGBM) algorithm and adds customizable
+constraints for group-wise fairness (_e.g._, equal opportunity, predictive equality) and other global goals (_e.g._,
+specific Recall or FPR targets).
+
+## Install
+
+FairGBM can be installed from [PyPI](https://pypi.org/project/fairgbm/) <!-- TODO: FairGBM pypi link -->
+
+```pip install fairgbm```
+
+or from GitHub
+
+```
+git clone --recurse-submodules https://github.com/feedzai/fairgbm.git
+pip install fairgbm/python-package
+```
+
+## Examples
+
+A short Python code snippet:
+```python
+from fairgbm import FairGBMClassifier
+
+# Instantiate
+fairgbm_clf = FairGBMClassifier(
+    "constraint_type"="FNR",    # constraint on equal group-wise TPR (equal opportunity)
```

Comment:
Why the quotes on the parameter names?

Reply:
ups, this was copied from a dictionary :sweat_smile: 

thanks!